tests/test_buffer.py

```diff
@@ -598,6 +598,52 @@ def test_SimpleArray_broadcast_slice_ghost_md(self):
                 for k in range(4):
                     self.assertEqual(ndarr2[i, j, k], sarr[i - G, j, k])
 
+    def test_SimpleArray_casting(self):
+        # first check the caster works if the argument is exact the same type
+        array_float64 = modmesh.SimpleArrayFloat64((2, 3, 4))
+        self.assertEqual(modmesh.testhelper.TestSimpleArrayHelper.
```

Comment:
Multi-lines like this look ugly.  Please beautify.

Reply:
@yungyuc  current 80 length limit of flake8 makes it difficult if any variable has a long name. 80 sounds too short for a mordern monitor, maybe consider to 120?

Reply:
It brings more pain to violate PEP-8.  We should stick with it.